core/runtime/src/main/java/io/quarkus/runtime/logging/PatternFormatterWithBanner.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.runtime.logging;
+
+import java.util.logging.Handler;
+
+import org.jboss.logmanager.formatters.PatternFormatter;
+
+public class PatternFormatterWithBanner extends PatternFormatter {
+    private String banner;
```

Comment:
This should be a `private final Supplier<String> bannerSupplier`, probably.

Reply:
done